src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1776,18 +1765,12 @@ public function getWebhookUrl (WC_Order $order)
 
 	    $webhook_url = WC()->api_request_url( strtolower( get_class( $this ) ) );
 	    $webhook_url = $this->removeTrailingSlashAfterParamater( $webhook_url );
-
-	    if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
-		    $webhook_url = add_query_arg(array(
-			    'order_id' => $order->id,
-			    'key'      => $order->order_key,
-		    ), $webhook_url);
-	    } else {
-		    $webhook_url = add_query_arg(array(
-			    'order_id' => $order->get_id(),
-			    'key'      => $order->get_order_key(),
-		    ), $webhook_url);
-	    }
+        $order_id = wooCommerceOrderId($order);
+        $order_key = wooCommerceOrderKey($order);
+        $webhook_url = add_query_arg(array(
+                                         'order_id' => $order_id,
+                                         'key'      => $order_key,
+                                     ), $webhook_url);
```

Comment:
Sounds like we have opportunity to make the code more DRY, why not put this into a internal method and reuse it?